ttter/settings.py

```diff
@@ -0,0 +1,128 @@
+"""
+Django settings for ttter project.
+
+Generated by 'django-admin startproject' using Django 4.0.2.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/4.0/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/4.0/ref/settings/
+"""
+
+from pathlib import Path
+
+# Build paths inside the project like this: BASE_DIR / 'subdir'.
+BASE_DIR = Path(__file__).resolve().parent.parent
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/4.0/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = 'django-insecure-p7nc14kk@9ifynf9*4&#dd=lnt0m24_cx%owh++e=_=qnu8&69'
+
+# SECURITY WARNING: don't run with debug turned on in production!
+DEBUG = True
+
+ALLOWED_HOSTS = []
+
+
+# Application definition
+
+INSTALLED_APPS = [
+    'django.contrib.admin',
+    'django.contrib.auth',
+    'django.contrib.contenttypes',
+    'django.contrib.sessions',
+    'django.contrib.messages',
+    'django.contrib.staticfiles',
+    'accounts',
+    'base',
+]
+
+MIDDLEWARE = [
+    'django.middleware.security.SecurityMiddleware',
+    'django.contrib.sessions.middleware.SessionMiddleware',
+    'django.middleware.common.CommonMiddleware',
+    'django.middleware.csrf.CsrfViewMiddleware',
+    'django.contrib.auth.middleware.AuthenticationMiddleware',
+    'django.contrib.messages.middleware.MessageMiddleware',
+    'django.middleware.clickjacking.XFrameOptionsMiddleware',
+]
+
+ROOT_URLCONF = 'ttter.urls'
+
+TEMPLATES = [
+    {
+        'BACKEND': 'django.template.backends.django.DjangoTemplates',
+        'DIRS': [BASE_DIR/'ttter'/'templates'],
```

Comment:
```suggestion
TEMPLATES = [
    {
        'BACKEND': 'django.template.backends.django.DjangoTemplates',
        'DIRS': [os.path.join(BASE_DIR, 'templates')],
```

'DIRS'の設定はos.path.join関数を通して設定してください。このように ' / ' で行うと、Windows環境ではうまく動作しなくなります。os.path.joinはos間での記述の違いを無視し、どのosでも動作できるようにするものです。os.path.joinを用いるためには、`import os`を記載する必要があるので、記載し忘れないようにしてください。

また、templatesのディレクトリ構成は基本的に、templatesディレクトリはプロジェクト直下に置き、その中に各アプリケーションディレクトリを入れる構成で作成するのが良いです。

ですので、ttterディレクトリの下にtemplatesディレクトリを置かず、プロジェクトディレクトリ直下、すなわちttterディレクトリやaccountsディレクトリと同じディレクトリの場所に配置し直してください。

Reply:
"templatesディレクトリはプロジェクト直下に置き、その中に各アプリケーションディレクトリを入れる構成"

ここの部分が誤解を与えてしまいそうなので以下に訂正します。

templatesディレクトリはプロジェクト直下に置き、その中に各アプリケーションの`名前と同じ`ディレクトリを作成してその中にhtmlファイルを入れる構成にしてください。

アプリケーションディレクトリを入れるわけではないです。誤解を招く表現で申し訳ないです。

Reply:
ttter
----accounts
----base
----static
      ----css
----ttter
     ----template
           ----base
           ----registration
           ↓
ttter
----accounts
----base
----static
      ----css
----ttter
----template
      ----base
      ----accounts
このような形でしょうか？

Reply:
ttter
----accounts
       -- views.pyなどのファイルがある
----base
      -- views.pyなどのファイルがある
----static
　　--css
----ttter
      -- settings.pyなどのファイルがある
----templates
　　--base
           -- この中にbaseに適用するhtmlファイル
　　--accounts
           --  この中にaccountsに適用するhtmlファイル

このような感じにしていただければ大丈夫です。

templatesディレクトリとstaticディレクトリは、os.path.join(BASE_DIR, '~ ') としているので、BASE_DIRはsettingsの親の親を見る、settings.pyにおける親の親は、ttter/ttter/settings.pyより、一番上のttterというディレクトリを指すことになるので、この直下に置くことで参照できるようになります。

Reply:
すいません　間隔開けたつもりだったのですが、自動的に詰まってしまうんですね。
ありがとうございます。